docker/README.md

```diff
@@ -327,6 +327,28 @@ You should now be able to configure [Jetpack Backup & Scan](https://jetpack.com/
 - Server password: `wordpress`
 - WordPress installation path: `/var/www/html`
 
+## Custom plugins & themes in the container
+
+Jetpack Docker environment can be wonderful for developing your own plugins and themes, too.
+
+Since everything under `mu-plugins` and `wordpress/wp-content` are git-ignored, you'll want to keep those folders outside Jetpack repository folder and link them as volumes to your Docker instance.
+
+1. First ensure your containers are stopped (`yarn docker:stop`).
+1. Create a docker-compose file. You can place it anywhere in your computer:
+	```yml
+	version: '3.3'
+	services:
+	  wordpress:
+	    volumes:
+	      - ~/my-plugin:/var/www/html/wp-content/plugins/my-plugin
+	```
+1. Start containers and include your custom volumes by running:
+	```bash
+	yarn docker:compose -f ~/docker-compose.my-volumes.yml up
```

Comment:
In this case, it will start the Jetpack containers and my own additional one as well? I don't need to specify the Jetpack one there?

Reply:
Yep, this is same as `yarn docker:up` which spins up all the containers. We just want to use the underlying `yarn docker:compose` command instead because we need to pass `-f` attribute _before_ `up` command. Ngrok command does something similar.

Reply:
>and my own additional one as well

So to clarify; no additional containers. This will _expand_ `volumes` list in existing `wordpress` container because Docker-compose combines multiple configs into one.